pywren/executor.py

```diff
@@ -85,18 +74,17 @@ def invoke_with_keys(self, s3_func_key, s3_data_key, s3_output_key,
             random.seed()
             runtime_url = random.choice(self.runtime_meta_info['urls'])
 
-
-        arg_dict = {'func_key' : s3_func_key,
-                    'data_key' : s3_data_key,
-                    'output_key' : s3_output_key,
-                    'status_key' : s3_status_key,
+        arg_dict = {'storage_info' : self.storage.get_storage_info(),
```

Comment:
This is the main thing that we discussed the other day -- can we see if there a better way to serialize / send information from the driver to the workers ? One possibility could be that we send a `config` object or the result of `extract_storage_config` and then recreate the StorageHandler on the worker side ?